HomeTask20/HomeTask20/Waiting.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using OpenQA.Selenium;
+using SeleniumExtras.WaitHelpers;
+using OpenQA.Selenium.Support.UI;
+
+namespace HomeTask20
+{
+    class Waiting
+    {
+        public static void WaitTheElement(IWebDriver driver, string xPath, int time = 5)
```

Comment:
Use one instance of WebDriverWait here. It's enough to share one instance for different cases